randutil/rand.go

```diff
@@ -32,34 +34,72 @@ import (
 	"time"
 )
 
-func init() {
-	// golang does not init Seed() itself
-	rand.Seed(time.Now().UnixNano() + int64(os.Getpid()))
+var defaultSeeder = SeedDatePid
+
+// SeedDatePid provides a basic seed value based only on
+// time and os PID value.
+func SeedDatePid() int64 {
```

Comment:
I don't love this name. It's awkward, and I think it should say what it's for rather than how it's implemented. Perhaps `AutoSeed`?

Reply:
Could ```DefaultSeeder``` work for you? It is the seeder that will be used by default if ```nil``` is passed in to NewPseudoRand.

Reply:
Yeah, that sounds good to me!